src/biblenotify/core/notifications.py

```diff
@@ -78,7 +78,7 @@ def loadVerses(self) -> dict:
 
         verses = json.loads(versesString)
         # Choose a random verse
-        self.currentVerse = verses["all"][random.randint(0, len(verses["all"]))]
+        self.currentVerse = verses["all"][random.randint(0, len(verses["all"]) - 1)]
```

Comment:
This prevents the random int range from going outside of the actual list rage because indexes start at 0.